### PR TITLE
List all profiles

### PR DESCRIPTION
**Pull request by jasonfiammetta**
### What this PR does (required):
Adds the UI required to display multiple profiles at once on the Dashboard page
Addresses Issue #40 

### Screenshots / Videos (front-end only):
Screen Shot 2022-01-17 at 2 01 03 PM

### Any information needed to test this feature (required):
- Log in and the ```/dashboard``` page should be the first thing you see

### CODE REVIEW

**Review by sundayezeilo — COMMENTED** [review]
This one looks good!

File: client/src/components/ProfileCard/ProfileCard.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import { Profile } from '../../interface/Profile';
+import ProfilePhoto from '../ProfilePhoto/ProfilePhoto';
+
+interface ProfileCardProps {
+  profile: Profile;
+}
+
+const ProfileCard: React.FC<ProfileCardProps> = ({ profile }) => {
+  const addRating = (newValue: number) => {
+    console.log('rated', newValue);
```
Comment (moffatethan) [inline client/src/components/ProfileCard/ProfileCard.tsx:13]: Note: I'm okay with this as it's going to be valuable for integration.

File: client/src/interface/Profile.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export interface Profile {
+  userId: string;
+  name: string;
+  occupation?: string;
+  rating?: number;
```
Comment (moffatethan) [inline client/src/interface/Profile.ts:5]: Good - when we finish work on separating the two models out we'll actually create a new interface for PetSitter and the rating would be more suited for there.

File: client/src/pages/Dashboard/Dashboard.tsx [hunk 3]
```diff
@@ -24,11 +27,32 @@ export default function Dashboard(): JSX.Element {
...
+            <ProfileCard profile={profile} />
+          </Grid>
+        ))}
+        <Grid item xs={12} textAlign="center">
+          <Button
+            size="large"
+            variant="outlined"
+            onClick={() => {
+              console.log('show more');
+            }}
```
Comment (moffatethan) [inline client/src/pages/Dashboard/Dashboard.tsx]: We can just leave the click handler off for now, we'll do this for pagination when the time comes.

**Review by moffatethan — CHANGES_REQUESTED** [review]
Really nice work here Jason, thanks for doing that handler for when we integrate reviews. Just a minor comment to address after should be good to merge in.

File: client/src/pages/Dashboard/Dashboard.tsx (hunk 3 above)
Comment (jasonfiammetta) [reply client/src/pages/Dashboard/Dashboard.tsx]: Sure Ethan, I'll take it off. I'm curious, is there some kind of standard to mark things for later? Maybe I should put in a ```// TODO``` instead? This goes to your other comment as well, considering that console log was useful for integration

Comment (moffatethan) [reply client/src/pages/Dashboard/Dashboard.tsx]: I think just as a team we'll know it's for integration 😄

**Review by moffatethan — APPROVED** [review]

**Pull request merged**